### DISCUSSION TITLE
Pytorch 教程翻译【持续迭代 - 2.x 最新版本】

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#622] [Announcements]
## 状态表示

✅ 表示已经完成
❌ 表示无人认领
‼️ 表示正在进行

> ✅ ["PyTorch Recipes"]()

*   [See All Recipes](https://pytorch.org/tutorials/recipes/recipes_index.html)
*   [See All Prototype Recipes](https://pytorch.org/tutorials/prototype/prototype_index.html)

> ✅ ["Introduction to PyTorch"]()

*   [Learn the Basics](https://pytorch.org/tutorials/beginner/basics/intro.html)
*   [Quickstart](https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html)
*   [Tensors](https://pytorch.org/tutorials/beginner/basics/tensorqs_tutorial.html)
*   [Datasets & DataLoaders](https://pytorch.org/tutorials/beginner/basics/data_tutorial.html)
*   [Transforms](https://pytorch.org/tutorials/beginner/basics/transforms_tutorial.html)
*   [Build the Neural Network](https://pytorch.org/tutorials/beginner/basics/buildmodel_tutorial.html)
*   [Automatic Differentiation with `torch.autograd`](https://pytorch.org/tutorials/beginner/basics/autogradqs_tutorial.html)
*   [Optimizing Model Parameters](https://pytorch.org/tutorials/beginner/basics/optimization_tutorial.html)
*   [Save and Load the Model](https://pytorch.org/tutorials/beginner/basics/saveloadrun_tutorial.html)

2.0版本翻译者： [Daydaylight](https://github.com/Daydaylight)

> ✅ ["Introduction to PyTorch on YouTube"]() 

*   [Introduction to PyTorch - YouTube Series](https://pytorch.org/tutorials/beginner/introyt.html)
*   [Introduction to PyTorch](https://pytorch.org/tutorials/beginner/introyt/introyt1_tutorial.html)
*   [Introduction to PyTorch Tensors](https://pytorch.org/tutorials/beginner/introyt/tensors_deeper_tutorial.html)
*   [The Fundamentals of Autograd](https://pytorch.org/tutorials/beginner/introyt/autogradyt_tutorial.html)
*   [Building Models with PyTorch](https://pytorch.org/tutorials/beginner/introyt/modelsyt_tutorial.html)
*   [PyTorch TensorBoard Support](https://pytorch.org/tutorials/beginner/introyt/tensorboardyt_tutorial.html)
*   [Training with PyTorch](https://pytorch.org/tutorials/beginner/introyt/trainingyt.html)
*   [Model Understanding with Captum](https://pytorch.org/tutorials/beginner/introyt/captumyt.html)

2.0版本翻译者：[Fadegentle](https://github.com/Fadegentle)

> ‼️ ["Learning PyTorch"]()

*  ‼️ [Deep Learning with PyTorch: A 60 Minute Blitz](https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html)
*  ✅ [Learning PyTorch with Examples](https://pytorch.org/tutorials/beginner/pytorch_with_examples.html)
*  ✅ [What is torch.nn *really*?](https://pytorch.org/tutorials/beginner/nn_tutorial.html)
*  ‼️ [Visualizing Models, Data, and Training with TensorBoard](https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html)

> ❌ ["Image and Video"]()

*   [TorchVision Object Detection Finetuning Tutorial](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html)
*   [Transfer Learning for Computer Vision Tutorial](https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html)
*   [Adversarial Example Generation](https://pytorch.org/tutorials/beginner/fgsm_tutorial.html)
*   [DCGAN Tutorial](https://pytorch.org/tutorials/beginner/dcgan_faces_tutorial.html)
*   [Spatial Transformer Networks Tutorial](https://pytorch.org/tutorials/intermediate/spatial_transformer_tutorial.html)
*   [Optimizing Vision Transformer Model for Deployment](https://pytorch.org/tutorials/beginner/vt_tutorial.html)

> ❌ ["Audio"]()

*   [Audio I/O](https://pytorch.org/tutorials/beginner/audio_io_tutorial.html)
*   [Audio Resampling](https://pytorch.org/tutorials/beginner/audio_resampling_tutorial.html)
*   [Audio Data Augmentation](https://pytorch.org/tutorials/beginner/audio_data_augmentation_tutorial.html)
*   [Audio Feature Extractions](https://pytorch.org/tutorials/beginner/audio_feature_extractions_tutorial.html)
*   [Audio Feature Augmentation](https://pytorch.org/tutorials/beginner/audio_feature_augmentation_tutorial.html)
*   [Audio Datasets](https://pytorch.org/tutorials/beginner/audio_datasets_tutorial.html)
*   [Speech Recognition with Wav2Vec2](https://pytorch.org/tutorials/intermediate/speech_recognition_pipeline_tutorial.html)
*   [Text-to-speech with Tacotron2](https://pytorch.org/tutorials/intermediate/text_to_speech_with_torchaudio.html)
*   [Forced Alignment with Wav2Vec2](https://pytorch.org/tutorials/intermediate/forced_alignment_with_torchaudio_tutorial.html)

> ❌ ["Text"]()

*   [Language Modeling with `nn.Transformer` and torchtext](https://pytorch.org/tutorials/beginner/transformer_tutorial.html)
*   [Fast Transformer Inference with Better Transformer](https://pytorch.org/tutorials/beginner/bettertransformer_tutorial.html)
*   [NLP From Scratch: Classifying Names with a Character-Level RNN](https://pytorch.org/tutorials/intermediate/char_rnn_classification_tutorial.html)
*   [NLP From Scratch: Generating Names with a Character-Level RNN](https://pytorch.org/tutorials/intermediate/char_rnn_generation_tutorial.html)
*   [NLP From Scratch: Translation with a Sequence to Sequence Network and Attention](https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html)
*   [Text classification with the torchtext library](https://pytorch.org/tutorials/beginner/text_sentiment_ngrams_tutorial.html)
*   [Language Translation with `nn.Transformer` and torchtext](https://pytorch.org/tutorials/beginner/translation_transformer.html)
*   [Preprocess custom text dataset using Torchtext](https://pytorch.org/tutorials/beginner/torchtext_custom_dataset_tutorial.html)

> ‼️ ["Reinforcement Learning"]()

*   ✅ [Reinforcement Learning (DQN) Tutorial](https://pytorch.org/tutorials/intermediate/reinforcement_q_learning.html)
*   ‼️ [Reinforcement Learning (PPO) with TorchRL Tutorial](https://pytorch.org/tutorials/intermediate/reinforcement_ppo.html)
*   ✅ [Train a Mario-playing RL Agent](https://pytorch.org/tutorials/intermediate/mario_rl_tutorial.html)

> ✅ ["Deploying PyTorch Models in Production"]()

*   [Deploying PyTorch in Python via a REST API with Flask](https://pytorch.org/tutorials/intermediate/flask_rest_api_tutorial.html)
*   [Introduction to TorchScript](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html)
*   [Loading a TorchScript Model in C++](https://pytorch.org/tutorials/advanced/cpp_export.html)
*   [(optional) Exporting a Model from PyTorch to ONNX and Running it using ONNX Runtime](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html)
*   [Real Time Inference on Raspberry Pi 4 (30 fps!)](https://pytorch.org/tutorials/intermediate/realtime_rpi.html)

2.0版本翻译者： [masteryi-0018](https://github.com/masteryi-0018)

> ✅ ["Code Transforms with FX"]()

*   [(beta) Building a Convolution/Batch Norm fuser in FX](https://pytorch.org/tutorials/intermediate/fx_conv_bn_fuser.html)
*   [(beta) Building a Simple CPU Performance Profiler with FX](https://pytorch.org/tutorials/intermediate/fx_profiling_tutorial.html)

> ❌ ["Frontend APIs"]()

*   [(beta) Channels Last Memory Format in PyTorch](https://pytorch.org/tutorials/intermediate/memory_format_tutorial.html)
*   [Forward-mode Automatic Differentiation (Beta)](https://pytorch.org/tutorials/intermediate/forward_ad_usage.html)
*   [Jacobians, Hessians, hvp, vhp, and more: composing function transforms](https://pytorch.org/tutorials/intermediate/jacobians_hessians.html)
*   [Model ensembling](https://pytorch.org/tutorials/intermediate/ensembling.html)
*   [Per-sample-gradients](https://pytorch.org/tutorials/intermediate/per_sample_grads.html)
*   [Using the PyTorch C++ Frontend](https://pytorch.org/tutorials/advanced/cpp_frontend.html)
*   [Dynamic Parallelism in TorchScript](https://pytorch.org/tutorials/advanced/torch-script-parallelism.html)
*   [Autograd in C++ Frontend](https://pytorch.org/tutorials/advanced/cpp_autograd.html)

> ❌ ["Extending PyTorch"]()

*   [Double Backward with Custom Functions](https://pytorch.org/tutorials/intermediate/custom_function_double_backward_tutorial.html)
*   [Fusing Convolution and Batch Norm using Custom Function](https://pytorch.org/tutorials/intermediate/custom_function_conv_bn_tutorial.html)
*   [Custom C++ and CUDA Extensions](https://pytorch.org/tutorials/advanced/cpp_extension.html)
*   [Extending TorchScript with Custom C++ Operators](https://pytorch.org/tutorials/advanced/torch_script_custom_ops.html)
*   [Extending TorchScript with Custom C++ Classes](https://pytorch.org/tutorials/advanced/torch_script_custom_classes.html)
*   [Registering a Dispatched Operator in C++](https://pytorch.org/tutorials/advanced/dispatcher.html)
*   [Extending dispatcher for a new backend in C++](https://pytorch.org/tutorials/advanced/extend_dispatcher.html)

> ❌ ["Model Optimization"]()

*   ‼️ [Profiling your PyTorch Module](https://pytorch.org/tutorials/beginner/profiler.html)
*   [PyTorch Profiler With TensorBoard](https://pytorch.org/tutorials/intermediate/tensorboard_profiler_tutorial.html)
*   [Hyperparameter tuning with Ray Tune](https://pytorch.org/tutorials/beginner/hyperparameter_tuning_tutorial.html)
*   [Optimizing Vision Transformer Model for Deployment](https://pytorch.org/tutorials/beginner/vt_tutorial.html)
*   [Parametrizations Tutorial](https://pytorch.org/tutorials/intermediate/parametrizations.html)
*   [Pruning Tutorial](https://pytorch.org/tutorials/intermediate/pruning_tutorial.html)
*   [(beta) Dynamic Quantization on an LSTM Word Language Model](https://pytorch.org/tutorials/advanced/dynamic_quantization_tutorial.html)
*   [(beta) Dynamic Quantization on BERT](https://pytorch.org/tutorials/intermediate/dynamic_quantization_bert_tutorial.html)
*   [(beta) Quantized Transfer Learning for Computer Vision Tutorial](https://pytorch.org/tutorials/intermediate/quantized_transfer_learning_tutorial.html)
*   [(beta) Static Quantization with Eager Mode in PyTorch](https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html)
*   [Grokking PyTorch Intel CPU performance from first principles](https://pytorch.org/tutorials/intermediate/torchserve_with_ipex.html)
*   [Grokking PyTorch Intel CPU performance from first principles (Part 2)](https://pytorch.org/tutorials/intermediate/torchserve_with_ipex_2.html)
*   [Getting Started - Accelerate Your Scripts with nvFuser](https://pytorch.org/tutorials/intermediate/nvfuser_intro_tutorial.html)
*   [Multi-Objective NAS with Ax](https://pytorch.org/tutorials/intermediate/ax_multiobjective_nas_tutorial.html)
*   [torch.compile Tutorial](https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html)
*   [Inductor CPU backend debugging and profiling](https://pytorch.org/tutorials/intermediate/inductor_debug_cpu.html)
*   [(Beta) Implementing High-Performance Transformers with Scaled Dot Product Attention (SDPA)](https://pytorch.org/tutorials/intermediate/scaled_dot_product_attention_tutorial.html)
*   [Using SDPA with `torch.compile`](https://pytorch.org/tutorials/intermediate/scaled_dot_product_attention_tutorial.html#using-sdpa-with-torch-compile)
*   [Conclusion](https://pytorch.org/tutorials/intermediate/scaled_dot_product_attention_tutorial.html#conclusion)

> ❌ ["Parallel and Distributed Training"]()

*   [Distributed and Parallel Training Tutorials](https://pytorch.org/tutorials/distributed/home.html)
*   [PyTorch Distributed Overview](https://pytorch.org/tutorials/beginner/dist_overview.html)
*   [Distributed Data Parallel in PyTorch - Video Tutorials](https://pytorch.org/tutorials/beginner/ddp_series_intro.html)
*   [Single-Machine Model Parallel Best Practices](https://pytorch.org/tutorials/intermediate/model_parallel_tutorial.html)
*   [Getting Started with Distributed Data Parallel](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html)
*   [Writing Distributed Applications with PyTorch](https://pytorch.org/tutorials/intermediate/dist_tuto.html)
*   [Getting Started with Fully Sharded Data Parallel(FSDP)](https://pytorch.org/tutorials/intermediate/FSDP_tutorial.html)
*   [Advanced Model Training with Fully Sharded Data Parallel (FSDP)](https://pytorch.org/tutorials/intermediate/FSDP_adavnced_tutorial.html)
*   [Customize Process Group Backends Using Cpp Extensions](https://pytorch.org/tutorials/intermediate/process_group_cpp_extension_tutorial.html)
*   [Getting Started with Distributed RPC Framework](https://pytorch.org/tutorials/intermediate/rpc_tutorial.html)
*   [Implementing a Parameter Server Using Distributed RPC Framework](https://pytorch.org/tutorials/intermediate/rpc_param_server_tutorial.html)
*   [Distributed Pipeline Parallelism Using RPC](https://pytorch.org/tutorials/intermediate/dist_pipeline_parallel_tutorial.html)
*   [Implementing Batch RPC Processing Using Asynchronous Executions](https://pytorch.org/tutorials/intermediate/rpc_async_execution.html)
*   [Combining Distributed DataParallel with Distributed RPC Framework](https://pytorch.org/tutorials/advanced/rpc_ddp_tutorial.html)
*   [Training Transformer models using Pipeline Parallelism](https://pytorch.org/tutorials/intermediate/pipeline_tutorial.html)
*   [Training Transformer models using Distributed Data Parallel and Pipeline Parallelism](https://pytorch.org/tutorials/advanced/ddp_pipeline.html)
*   [Distributed Training with Uneven Inputs Using the Join Context Manager](https://pytorch.org/tutorials/advanced/generic_join.html)

> ❌ ["Mobile"]()

*   [Image Segmentation DeepLabV3 on iOS](https://pytorch.org/tutorials/beginner/deeplabv3_on_ios.html)
*   [Image Segmentation DeepLabV3 on Android](https://pytorch.org/tutorials/beginner/deeplabv3_on_android.html)

> ✅ ["Recommendation Systems"]()

*   [Introduction to TorchRec](https://pytorch.org/tutorials/intermediate/torchrec_tutorial.html)
*   [Exploring TorchRec sharding](https://pytorch.org/tutorials/advanced/sharding.html)

> ✅ ["Multimodality"]()

*   [TorchMultimodal Tutorial: Finetuning FLAVA](https://pytorch.org/tutorials/beginner/flava_finetuning_tutorial.html)

**Comment by fangxiaoshen** [comment]
["Recommendation Systems"](https://github.com/apachecn/pytorch-doc-zh/discussions/622)

[Introduction to TorchRec](https://pytorch.org/tutorials/intermediate/torchrec_tutorial.html)
[Exploring TorchRec sharding](https://pytorch.org/tutorials/advanced/sharding.html)

**Reply by jiangzhonglian** [reply]
好兄弟，记得在这里登记哈： https://github.com/apachecn/pytorch-doc-zh/discussions/583
